R/plotly_POST.R

```diff
@@ -55,6 +72,14 @@ plotly_POST <- function(x, filename, fileopt = "new", world_readable = TRUE) {
   base_url <- file.path(get_domain(), "clientresp")
   resp <- httr::POST(base_url, body = bod)
   con <- process(struct(resp, "clientresp"))
+  if (sharing[1] == "secret") {
+    bits <- strsplit(con$url, "/")[[1]]
+    plot_id <- bits[length(bits)]
+    url <- paste0(get_domain("v2"), "files/", verify("username"), ":", plot_id)
+    bod <- list(share_key_enabled = TRUE)
+    con2 <- httr::PATCH(url, plotly_headers("v2"), body = bod, encode = "json")
+    con$url <- paste0(con$url, "?share_key=", content(con2)$share_key)
```

Comment:
@chriddyp is there anything else from the `PATCH` response that you'd like to return?